src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -14352,7 +14352,7 @@ void PrintHelp (__in_z LPCSTR pszCmdName)
             return;
         }
 #else
-        int err = PAL_InitializeDLL();
+        int err = PAL_InitializeDLL(TRUE);
```

Comment:
Is there a reason why it should not just take the `PAL_INITIALIZE_` flags on what to initialize as an argument"?